src/main/java/uk/gov/hmcts/reform/et/syaapi/models/AcasCertificate.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.et.syaapi.models;
+
+import java.util.Objects;
+
+/**
+ * Simply holds the certificate blob data as a string (in Base64 encoded format) along with the associated ACAS number.
+ */
+public class AcasCertificate {
+
+    private String certificateDocument;
+    private String certificateNumber;
+
+    public String getCertificateNumber() {
```

Comment:
Can we replace this getters and setters by making this class as @Data ? So that it will behave as POJO.

Reply:
Updated